src/heap.c

```diff
@@ -1,26 +1,27 @@
 #include "common.h"
 
-extern s32 D_80268000;
-extern s32 D_802FB800;
+extern HeapNode heap_collisionHead; //D_80268000
+extern HeapNode heap_generalHead; //D_802FB800
+extern HeapNode heap_battleHead; //D_803DA800
```

Comment:
Remove these old comments

Reply:
Oops, left them in when doing the mass renaming so I didn't mix them up.